### DISCUSSION TITLE
Draft/publication status should be scoped to locale

### DISCUSSION THREAD

**Discussion by ssyberg** [payloadcms/payload#2577] [Feature Requests & Ideas]
If a translation is in progress having it in a saved draft state while the other locales are published _and updateable_ would be ideal.

**Comment by hotrungnhan** [comment]
Locale should able to be list && switch in collection editor instead of switch all the UI...
Better , If the locale switcher pannel can show the status in percent of fields that are filled

**Reply by qbeauperin** [reply]
> Better , If the locale switcher pannel can show the status in percent of fields that are filled

I made a little plugin for this. If it still makes sense after Payload's admin UX improvements in the next couple of months (#1234), I can clean it up and release it as a package. 😊 

![image](https://github.com/payloadcms/payload/assets/1763338/08e537d6-d7af-4459-9213-0684b1c9c4e2)

**Comment by MarvinVrdoljak** [comment]
This is something we'd need as well. Any news here?

**Reply by ssyberg** [reply]
Yea this is an absolute requirement for us, we're going to end up just building it out I think.

**Comment by ssyberg** [comment]
cc @manuelebagnolini not sure if you guys are considering publishing state and percent translated in your plugin ☝🏼

**Reply by manuelebagnolini** [reply]
I've talked to the development team about it and we think it's an interesting idea to implement in future versions 👍

**Comment by ssyberg** [comment]
@DanRibbens did this end up happening in 2.0? I don't see mention of it in the docs.

**Reply by jhnsml** [reply]
It's not in 2.0. Was supposed to go in 2.1.0 from what @DanRibbens mentioned but I can't see it. 🥲

**Reply by qbeauperin** [reply]
FIY, @jmikrut responded [on Discord](https://discord.com/channels/967097582721572934/1102950643259424828/1172187844815622154):
> this is actively in the design phase and honestly there's not much development work necessary so i think it'll probably be coming in like a month or so

**Comment by JessRynkar** [comment]
Linking to our [roadmap item](https://github.com/payloadcms/payload/discussions/4194), the work for this is underway!
